src/main/java/mate/academy/hwspringbootintro/dto/order/OrderSetStatusRequestDto.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.hwspringbootintro.dto.order;
+
+import mate.academy.hwspringbootintro.model.Order;
+
+public record OrderSetStatusRequestDto(Order.Status status) {
```

Comment:
should we validate request?
```suggestion
public record UpdateOrderRequestDto(Order.Status status) {
```